lms/djangoapps/onboarding/models.py

```diff
@@ -582,73 +569,35 @@ def get_user_hear_about_philanthropy(self, _type="labels"):
         :return: list of labels / names of fields
         """
         if _type == "labels":
```

Comment:
single qoutes, here and in the function prototype and all such instances in the file please.